Resources/Private/source.js

```diff
@@ -135,7 +135,8 @@ window.addEventListener('DOMContentLoaded', () => {
 
 		const multiple = container.dataset.assetMultiple !== undefined;
 		const assetIdentifier = container.dataset.asset;
-		const url = assetIdentifier && !multiple ? new URL('/neos/media/browser/images/edit.html?asset=' + assetIdentifier, window.location.origin) : new URL('/neos/media/browser/assets/index.html', window.location.origin);
+		const mediaBrowserUriPath = container.dataset.assetMediaBrowserUriPath || '/neos/media/browser';
+		const url = assetIdentifier && !multiple ? new URL(mediaBrowserUriPath + '/images/edit.html?asset=' + assetIdentifier, window.location.origin) : new URL(mediaBrowserUriPath + '/assets/index.html', window.location.origin);
```

Comment:
Looking at the code of this plugin: If we respect the (potential) subfolder here. Isn't it missing in line 39?

Reply:
The comment is now marked outdated but there are only two commits from 1 h ago!? @so-grimm did you force-push to the branch?

Reply:
This comment was made on the first commit. The second commit did change these lines ... Shouldn't they be outdated now?